staging/src/k8s.io/apimachinery/pkg/runtime/scheme.go

```diff
@@ -129,12 +129,6 @@ func (s *Scheme) nameFunc(t reflect.Type) string {
 	return gvks[0].Kind
 }
 
-// fromScope gets the input version, desired output version, and desired Scheme
```

Comment:
Please move this commit to a separate PR.

Reply:
i have moved it.